guessNumb.js

```diff
@@ -0,0 +1,84 @@
+//--------------------------READLINE------------------------------//
+
+const readline = require("readline");
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    rl.question(questionText, resolve);
+  });
+}
+
+//-------------------------AYSNC START----------------------------//
+
+start();
+
+async function start() {
+  let maxNum = 100;
+  let minNum = 0;
+  let firstGuess = 50;
+
+  //This function returns a random number between 2 given integers.
+
+  function randoNum(higher, lower) {
+    let range = higher - lower + 1;
+    return Math.floor(Math.random() * range) + lower;
+  }
+
+  //This function returns the median of 2 given integers.
+
+  function guessNum(max, min) {
+    return Math.floor((max - min) / 2 + min);
+  }
+
+  console.log(
+    "Let's play a game where you, my human friend, pick up a number 1-100 and I, Computer, try to guess it."
+  );
+
+  let secretNumber = await ask(
+    "What is your secret number?\nI won't peek, I promise...\n>_ "
+  );
+  console.log(
+    `You entered: ${secretNumber}.\nNo worries! this is just so i know you aren't cheating ;) .`
+  );
```

Comment:
For user experience, you may want to make clear the response the computer expects. As your code is written, this would be "yes" and "no". Same thing later on, with "higher" and "lower".